tests/test_ingest.py

```diff
@@ -29,6 +30,12 @@ def test_ingest_sessions(pipeline, sessions_csv, ingest_sessions):
     assert (session.SessionDirectory
             & {'subject': sess.name}).fetch1('session_dir') == sess.session_dir
 
+''' Delete these?
+CB: I think these tests are depreciated with the update to permit multiple root directories
+    Previously, they tested against known bad roots to make sure root/session matched
+    Now, we have to run find_full_path every on mult roots regardless.
+    To update would result in tautology:
+    > assert find_full_path == find_full_path
```

Comment:
```suggestion
```

In the `test_find_valid_full_path` function, the following should still allow for testing the `find_full_path` function by changing the line from:
```
session_full_path = pathlib.Path(get_ephys_root_data_dir()) / sess.session_dir
```

to:
```
session_full_path = pathlib.Path('/main/workflow-array-ephys/tests/user_data') / sess.session_dir
```